### PR TITLE
Refactoring hfss3dlayout.py Class

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

**Review by Samuelopez-ansys — DISMISSED** [review]
LGTM

File: pyaedt/hfss3dlayout.py [hunk 1]
```diff
@@ -490,7 +505,7 @@ def create_differential_port(self, via_signal, via_reference, port_name, deembed
             Signal pin.
         via_reference : float
             Reference pin.
-        port_name : str
+        name : str
             New Port Name.
         deembed : bool, optional
             Either to deembed parasitics or not. Default is `True`.
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:511]:
```suggestion
            Whether to deembed parasitics. The default is ``True``.
```

File: pyaedt/hfss3dlayout.py [hunk 2]
```diff
@@ -506,21 +521,21 @@ def create_differential_port(self, via_signal, via_reference, port_name, deembed
         >>> oEditor.CreateEdgePort
         """
         listp = self.port_list
-        if port_name in self.port_list:
-            self.logger.error("Port already existing on via {}".format(port_name))
+        if name in self.port_list:
+            self.logger.error("Port already existing on via {}".format(name))
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:525]:
```suggestion
            self.logger.error("Port already existd on via {}.".format(name))
```

File: pyaedt/hfss3dlayout.py [hunk 3]
```diff
@@ -529,13 +544,13 @@ def create_differential_port(self, via_signal, via_reference, port_name, deembed
         else:
             return False
 
-    @pyaedt_function_handler()
-    def create_coax_port(self, vianame, radial_extent=0.1, layer=None, alignment="lower"):
+    @pyaedt_function_handler(vianame="via")
+    def create_coax_port(self, via, radial_extent=0.1, layer=None, alignment="lower"):
         """Create a new coax port.
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:549]:
```suggestion
        """Create a coax port.
```

File: pyaedt/hfss3dlayout.py [hunk 4]
```diff
@@ -555,10 +570,10 @@ def create_coax_port(self, vianame, radial_extent=0.1, layer=None, alignment="lo
         >>> oEditor.CreateEdgePort
         """
         listp = self.port_list
-        if vianame in self.port_list:
-            self.logger.error("Port already existing on via {}".format(vianame))
+        if via in self.port_list:
+            self.logger.error("Port already existing on via {}".format(via))
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:574]:
```suggestion
            self.logger.error("Port already exists on via {}.".format(via))
```

File: pyaedt/hfss3dlayout.py [hunk 5]
```diff
@@ -739,7 +754,7 @@ def validate_full_design(self, name=None, outputdir=None, ports=None):
         # The design validation inside HFSS outputs to a separate log file which we merge into this overall file
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:754]:
```suggestion
        # The design validation inside HFSS outputs to a separate log file that is merged into this overall file
```

File: pyaedt/hfss3dlayout.py [hunk 6]
```diff
@@ -1313,20 +1332,20 @@ def import_dxf(self, dxf_path, aedb_path=None, control_file=None, set_as_active=
...
 
         Parameters
         ----------
-        gerber_path : str
+        input_file : str
             Full path to the Gerber zip file.
-        aedb_path : str, optional
-            Full path to the AEDB file.
+        output_dir : str, optional
+            Full path to the AEDB folder. Example ``"c:\\temp\\test.aedb"``
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:1348]:
```suggestion
            Full path to the AEDB folder. For example, ``"c:\\temp\\test.aedb"``.
```

File: pyaedt/hfss3dlayout.py [hunk 7]
```diff
@@ -1345,20 +1364,20 @@ def import_gerber(
...
         """Import a board file into HFSS 3D Layout and assign the stackup from an XML file if present.
 
         Parameters
         ----------
         input_file : str
             Full path to the board file.
-        aedb_path : str, optional
-            Full path to the AEDB file.
+        output_dir : str, optional
+            Full path to the AEDB folder. Example ``"c:\\temp\\test.aedb"``
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:1380]:
```suggestion
            Full path to the AEDB folder. For example, ``"c:\\temp\\test.aedb"``.
```

File: pyaedt/hfss3dlayout.py [hunk 8]
```diff
@@ -1378,20 +1397,20 @@ def import_brd(
...
         """Import an AWR Microwave Office file into HFSS 3D Layout and assign the stackup from an XML file if present.
 
         Parameters
         ----------
         input_file : str
             Full path to the AWR Microwave Office file.
-        aedb_path : str, optional
-            Full path to the AEDB file.
+        output_dir : str, optional
+            Full path to the AEDB folder. Example ``"c:\\temp\\test.aedb"``
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:1413]:
```suggestion
            Full path to the AEDB folder. For example, ``"c:\\temp\\test.aedb"``.
```

File: pyaedt/hfss3dlayout.py [hunk 9]
```diff
@@ -1410,20 +1429,20 @@ def import_awr(
...
         """Import an IPC2581 file into HFSS 3D Layout and assign the stackup from an XML file if present.
 
         Parameters
         ----------
         input_file : str
             Full path to the IPC2581 file.
-        aedb_path : str, optional
-            Full path to the AEDB file.
+        output_dir : str, optional
+            Full path to the AEDB folder. Example ``"c:\\temp\\test.aedb"``
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:1445]:
```suggestion
            Full path to the AEDB folder. For example, ``"c:\\temp\\test.aedb"``.
```

File: pyaedt/hfss3dlayout.py [hunk 10]
```diff
@@ -1442,18 +1461,20 @@ def import_ipc2581(
...
         """Import an ODB++ file into HFSS 3D Layout and assign the stackup from an XML file if present.
 
         Parameters
         ----------
         input_file : str
             Full path to the ODB++ file.
-        aedb_path : str, optional
-            Full path to the AEDB file.
+        output_dir : str, optional
+            Full path to the AEDB folder. Example ``"c:\\temp\\test.aedb"``
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:1477]:
```suggestion
            Full path to the AEDB folder. For example, ``"c:\\temp\\test.aedb"``.
```

File: pyaedt/hfss3dlayout.py [hunk 11]
```diff
@@ -1758,12 +1786,12 @@ def load_diff_pairs_from_file(self, filename):
         ----------
         >>> oModule.LoadDiffPairsFromFile
         """
-        if not os.path.isfile(filename):  # pragma: no cover
-            raise ValueError("{}: unable to find the specified file.".format(filename))
+        if not os.path.isfile(input_file):  # pragma: no cover
+            raise ValueError("{}: unable to find the specified file.".format(input_file))
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:1790]:
```suggestion
            raise ValueError("{}: Unable to find the specified file.".format(input_file))
```
Add concluding period. (I don't quite know where it goes.)

File: pyaedt/hfss3dlayout.py [hunk 12]
```diff
@@ -1775,16 +1803,16 @@ def load_diff_pairs_from_file(self, filename):
...
         return True
 
-    @pyaedt_function_handler()
-    def save_diff_pairs_to_file(self, filename):
+    @pyaedt_function_handler(filename="input_file")
+    def save_diff_pairs_to_file(self, input_file):
         # type: (str) -> bool
         """Save differtential pairs definition to a file.
 
         If a file with the specified name already exists, it is overwritten.
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:1811]: Same comment as earlier about possibly moving this to the description for the  ``input_file`` parameter.

File: pyaedt/hfss3dlayout.py [hunk 13]
```diff
@@ -1796,17 +1824,17 @@ def save_diff_pairs_to_file(self, filename):
...
-    def export_3d_model(self, file_name=None):
+    @pyaedt_function_handler(file_name="output_file")
+    def export_3d_model(self, output_file=None):
         """Export the Ecad model to a 3D file.
 
         Parameters
         ----------
-        file_name : str, optional
+        output_file : str, optional
             Full name of the file to export. The default is None, in which case the file name is
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:1838]:
```suggestion
            Full name of the file to export. The default is ``None``, in which case the file name is
```

File: pyaedt/hfss3dlayout.py [hunk 14]
```diff
@@ -1796,17 +1824,17 @@ def save_diff_pairs_to_file(self, filename):
...
             set to the design name and saved as a SAT file in the working directory.
             Extensions available are ``"sat"``, ``"sab"``, and ``"sm3"`` up to AEDT 2022R2 and
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:1840]:
```suggestion
            Extensions available are ``"sat"``, ``"sab"``, and ``"sm3"`` up to AEDT 2022 R2 and
```

File: pyaedt/hfss3dlayout.py [hunk 15]
```diff
@@ -1979,9 +2007,9 @@ def edit_source_from_file(
...
         Parameters
         ----------
-        source_name : str
+        source : str
             Source Name.
-        file_name : str
+        input_file : str
             Full name of the input file.
         is_time_domain : bool, optional
             Either if the input data is Time based or Frequency Based. Frequency based data are Mag/Phase (deg).
```
Comment (PipKat) [inline pyaedt/hfss3dlayout.py:2015]:
```suggestion
            Whether the input data is time-based. The defaulti s ``True``. If
            ``False``, the input data is frequency-based. Frequency-based data
            is degrees in this format: ``Mag/Phase``.
```

File: pyaedt/hfss3dlayout.py [hunk 16]
```diff
@@ -1735,18 +1760,18 @@ def get_differential_pairs(self):
...
 
         You can use the ``save_diff_pairs_to_file`` method to obtain the file format.
         The ``File End Of Line`` must be UNIX (LF).
         New definitions are added only if compatible with the existing definition already defined in the project.
 
         Parameters
         ----------
-        filename : str
+        input_file : str
             Fully qualified name of the file containing the differential pairs definition.
```
Comment (Samuelopez-ansys) [inline pyaedt/hfss3dlayout.py:1775]:
```suggestion
            Full path to the differential pairs definition file.
```

File: pyaedt/hfss3dlayout.py [hunk 17]
```diff
@@ -1775,16 +1800,16 @@ def load_diff_pairs_from_file(self, filename):
...
         # type: (str) -> bool
         """Save differtential pairs definition to a file.
 
         If a file with the specified name already exists, it is overwritten.
 
         Parameters
         ----------
-        filename : str
+        output_file : str
             Fully qualified name of the file containing the differential pairs definition.
```
Comment (Samuelopez-ansys) [inline pyaedt/hfss3dlayout.py:1813]:
```suggestion
            Full path to the differential pairs definition file.
```

**Review by Samuelopez-ansys — DISMISSED** [review]

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**